packages/react/tests/e2e-storybook/cypress/integration/Footer/Footer.e2e.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import settings from 'carbon-components/es/globals/js/settings';
+
+const { prefix } = settings;
+
+/**
+ * Sets the correct path (Micro language only)
+ *
+ * @type {string}
+ * @private
+ */
+
+const _pathMicroLanguageOnly =
+  '/iframe.html?id=components-footer--micro-language-only';
+
+describe('dds-footer | Micro language only (desktop)', () => {
+  beforeEach(() => {
+    cy.visit(`/${_pathMicroLanguageOnly}`);
+    cy.viewport(1280, 780);
+  });
+
+  it('should load language selector dropdown', () => {
+    cy.get(`[data-autoid="dds--language-selector"]`).click();
+
+    cy.screenshot();
+    cy.percySnapshot(
+      'dds-footer | Micro language only (desktop language selector)',
```

Comment:
can we use `Footer` instead of `dds-footer` for these React snapshots?

Reply:
updated the names!